dialogflow/stopwatch/Stopwatch/BottomDrawerViewControllers.swift

```diff
@@ -0,0 +1,154 @@
+    }
+  }
+}
+
+class DrawerContentViewController: UITableViewController {
+  var preferredHeight: CGFloat = 200
+  var menuItems = [BetaFeatureMenu.sentimentAnalysis,  BetaFeatureMenu.textToSpeech,
+                   BetaFeatureMenu.knowledgeConnector]
+
+  override var preferredContentSize: CGSize {
+    get {
+      return CGSize(width: view.bounds.width, height: preferredHeight)
+    }
+    set {
+      super.preferredContentSize = newValue
+    }
+  }
+
+  init() {
+    super.init(nibName: nil, bundle: nil)
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.tableView.register(UITableViewCell.self, forCellReuseIdentifier: "Cell")
+    tableView.separatorStyle = .none
+    tableView.allowsMultipleSelection = true
+  }
+
+  override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    let cell = tableView.dequeueReusableCell(withIdentifier: "Cell", for: indexPath)
+    cell.textLabel?.text = "\(menuItems[indexPath.item].stringValue())"
+    cell.tintColor = ApplicationScheme.shared.colorScheme.primaryColor
+    //cell.backgroundColor = colorScheme.surfaceColor
+    print(cell.textLabel?.text ?? "")
+    let defaults = UserDefaults.standard
+    if let defaultItems = defaults.value(forKey: Constants.selectedMenuItems) as? [Int],
+      defaultItems.count > 0 {
+      if defaultItems.contains(indexPath.row) {
+        cell.accessoryView = UIImageView(image: #imageLiteral(resourceName: "ic_done"))
+      } else {
+        cell.accessoryView = nil
+      }
+    }
+
+    return cell
+  }
+
+  override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return menuItems.count
+  }
+
+  override func numberOfSections(in tableView: UITableView) -> Int {
+    return 1
+  }
+
+  override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+    let cell = tableView.cellForRow(at: indexPath)
+
+    tableView.deselectRow(at: indexPath, animated: true)
+    let defaults = UserDefaults.standard
+    var items = [Int]()
+    if let defaultItems = defaults.value(forKey: Constants.selectedMenuItems) as? [Int],
+      defaultItems.count > 0 {
+      items = defaultItems
+    }
+
+    //        if let item = BetaFeatureMenu(rawValue: indexPath.row) {
+    if let index = items.firstIndex(of: indexPath.row) {
+      items.remove(at: index)
+      cell?.accessoryView = nil
+    } else {
+      items.append(indexPath.row)
+      cell?.accessoryView = UIImageView(image: #imageLiteral(resourceName: "ic_done"))
+    }
+    //        }
+    defaults.set(items, forKey: Constants.selectedMenuItems)
+  }
+
+}
+
+class DrawerHeaderViewController: UIViewController,MDCBottomDrawerHeader {
+  let preferredHeight: CGFloat = 80
+  let titleLabel : UILabel = {
+    let label = UILabel(frame: .zero)
+    label.text = "Optional Features"
```

Comment:
Please change this line to the following:
label.text = "Tap to enable additional Dialogflow features."